cli/aws_orbit/data/toolkit/template.yaml

```diff
@@ -196,15 +196,15 @@ Resources:
             Action:
               - redshift:*
             Resource:
-              - arn:aws:redshift:${region}:${account_id}:parametergroup:${top_level}-${env_name}-*
-              - arn:aws:redshift:${region}:${account_id}:securitygroup:${top_level}-${env_name}-*
-              - arn:aws:redshift:${region}:${account_id}:subnetgroup:${top_level}-${env_name}-*
-              - arn:aws:redshift:${region}:${account_id}:cluster:${top_level}-${env_name}-*
+              - arn:aws:redshift:${region}:${account_id}:parametergroup:orbit-${env_name}-*
+              - arn:aws:redshift:${region}:${account_id}:securitygroup:orbit-${env_name}-*
+              - arn:aws:redshift:${region}:${account_id}:subnetgroup:orbit-${env_name}-*
+              - arn:aws:redshift:${region}:${account_id}:cluster:${env_name}-*
```

Comment:
from what i can tell, this is the only change that should be required. the {top-level} is already being replaced with "orbit" for the env/teams toolkit role. but i do see where the cluster doesn't get a name prefixed with "orbit-" when it is created. that is why the delete using boto3 is failing. i feel a better fix would be to prefix the cluster name with "orbit-" and then this PR isn't needed. if we don't want to do that, then i think dropping the changes to lines 199, 200, 201, and 207 and only modifying the resource for the cluster is ok.